playbooks/openstack-ses_aio_instance.yml

```diff
@@ -0,0 +1,89 @@
+---
+- hosts: localhost
+  gather_facts: no
+  connection: local
+  roles:
+    - ansible-config_template
+  post_tasks:
+    - name: Load generic vars
+      include_vars: "{{ playbook_dir }}/../vars/common-vars.yml"
+    - name: Load openstack vars
+      include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
+
+    - name: Handle creation
+      when:
+        - not ((ses_node_delete | default('false'))| bool)
+      block:
```

Comment:
This block has no `rescue` so we can potentially fail to create the server and the volume will still be there with no cleanup

Reply:
Correct.